commands.md

```diff
@@ -33,3 +33,4 @@ Command | Implemented By | Who can run it | Description
 `/release-note` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note` label
 `/release-note-action-required` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-action-required` label
 `/release-note-none` | prow [releasenote](./prow/plugins/releasenote) | authors and kubernetes org members | adds the `release-note-none` label
+`/status [label1 label2 ...]` | prow [label](./prow/plugins/label) | anyone | adds a status/<> label(s) if it exists
```

Comment:
Not anyone can use this command. Only `@kubernetes/kubernetes-milestone-maintainers` team members.